### ISSUE TITLE
5km mesh specification 

### ISSUE THREAD

**Issue by balbionthebox** [Kanahiro/jismesh-raster#6]
Thanks for this work Kanahiro
If you are willing to add the 5 km mesh too, we found a specification in this document, from page 7:
https://www.meti.go.jp/statistics/tyo/kougyo/mesh/2003/kaisetsu/pdf/gaiyo1.pdf

**Comment by Kanahiro** [comment]
Hi @balbionthebox ! Thank you for comment.
I just released `v0.3.1`, supporting 5km/2km mesh following the specification.
However, I don't have enough confidence it correctly works because I don't have 5km-mesh data.
Can you try this? And it is fotunate sample data  is shared if you could, Thanks.

**Comment by balbionthebox** [comment]
I did try with this data that is supposed to be a 5 km mesh ( 1= presence of bears) but get a similar outcome as before
[bears_presence.csv](https://github.com/Kanahiro/jismesh-raster/files/8989758/bears_presence.csv)

**Comment by Kanahiro** [comment]
hi @balbionthebox , sorry for late response.
I tried to make a mesh from your csv with following command.

```sh
python ./jismesh_raster/rasterize.py ./tests/fixture/bears_presence.csv mesh.tif --meshcol 1 --valuecol 3 --method sum
```

and output is 
![スクリーンショット (1)](https://user-images.githubusercontent.com/20744195/180585484-9b5bb6ae-637f-4a37-8f0f-ec79c3234e01.png)

`5kmMesh` col may not be a correct code for 5km mesh spec because 5km-meshcode is 7-figure.
So I selected `Descriptio` as meshcode col and got above output.
